lib/stchart/assets/table.html.erb

```diff
@@ -37,7 +37,7 @@
     <tr class='person' id='commits_<%=_short_id(id)%>' title='<%=_short_id(id)%>'>
       <td style="background-color: <%= color[0] %>;">&nbsp;</td>
       <td class='name'>
-        <a href="http://stackalytics.com/?user_id=<%=id%>">
+        <a href="http://stackalytics.com/?user_id=<%=id%>&metric=<%= metric == 'reviews' ? 'marks' : metric %>">
```

Comment:
Could you use "&amp;" instead of "&" ?

* http://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href